app/views/devise/registrations/new.html.erb

```diff
@@ -1,25 +1,44 @@
-<h2>Sign up</h2>
-
+<!-- This is the Sign up Landing Page -->
+<div class="container">
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= devise_error_messages! %>
 
-  <div class="field">
-    <%= f.label :username %><br />
-    <%= f.text_field :username, autofocus: true %>
-  </div>
+  <body>
```

Comment:
these body tags shouldn't be here. They are already in the `app/views/layouts/application.html.erb`.